src/python/createrepo_c/__init__.py

```diff
@@ -17,9 +17,12 @@
 
 UNKNOWN_CHECKSUM    = _createrepo_c.CHECKSUM_UNKNOWN #: Checksum unknown
 CHECKSUM_UNKNOWN    = _createrepo_c.CHECKSUM_UNKNOWN #: Checksum unknown
-MD5                 = _createrepo_c.MD5              #: MD5 checksum
-SHA                 = _createrepo_c.SHA              #: SHA1 checksum alias
-SHA1                = _createrepo_c.SHA1             #: SHA1 checksum
+if hasattr(_createrepo_c, 'MD5'):
+    MD5             = _createrepo_c.MD5              #: MD5 checksum
+if hasattr(_createrepo_c, 'SHA'):
+    SHA             = _createrepo_c.SHA              #: SHA1 checksum alias
+if hasattr(_createrepo_c, 'SHA1'):
+    SHA1            = _createrepo_c.SHA1             #: SHA1 checksum
```

Comment:
Couldn't this be made more future-proof?
```
for hash_name in ('MD5', 'SHA', 'SHA1', 'SHA224', 'SHA256', 'SHA512'):
    hash = getattr(_createrepo_c, hash_name, None)
    if hash:
        globals()[hash_name] = hash
```

Reply:
That is neat, thanks.